sky/shell/tracing_controller.cc

```diff
@@ -148,6 +150,23 @@ static void BaseTraceEventCallback(base::TraceTicks timestamp,
   }
 }
 
+static void TraceThreadMetadataToObservatory() {
+  const char* name = base::PlatformThread::GetName();
+  if (name == nullptr) {
+    return;
+  }
+  Dart_SetThreadName(name);
```

Comment:
No. We dont. Specifically the platform thread. But I don't know how to get a hold of the platform thread task runner from the shell or tracing controller. I could make these calls once when we initially set thread names but then I am not sure if `Dart_SetThreadName` can be made before `Dart_Initialize`. cc @johnmccutchan for clarification.